### DISCUSSION TITLE
Adding Kawaii Physics to certain animations

### DISCUSSION THREAD

**Discussion by TheJaaso** [pafuhana1213/KawaiiPhysics#146] [Q&A] [answered]
Same question as title.  Instead of applying it to the character wholesale, only have it happen on the animation themselves?  The reason being is that some of my animations have a ton of snap to them and I don't want the kawaii awesome happening in them but for when the character is recovering from them and/or moving around.

**Comment by pafuhana1213** [comment, marked as answer]
Hi @TheJaaso 
Kawaii itself does not have such a function, but you can probably meet your needs by controlling the Alpha of the AnimNode with properties or AnimCurve.

thanks!
